src/redis.c

```diff
@@ -394,6 +394,49 @@ static int redis_handle_info(char *node, char const *info_line,
   return 0;
 } /* }}} int redis_handle_info */
 
+static int redis_slave_info(char *node, char const *info_line) /* {{{ */
+{
+  char *saveptr = NULL;
+  char *str = strstr(info_line, "connected_slaves");
+  char slave[] = "slave";
+  if (str != NULL) {
```

Comment:
Hi, can I please ask you to reformat your code, to get fast return.

```
if (str == NULL) 
   return 0;

... rest of code...
```

That is preferred and more readable.
If that is possible in do { } block, please define variable on first use and not in the beginning of block.
https://github.com/collectd/collectd/blob/master/docs/review_comments.md

Thanks.